CMakeLists.txt

```
@@ -35,6 +35,8 @@ endif()
 option(SOLC_LINK_STATIC "Link solc executable statically on supported platforms" OFF)
 option(SOLC_STATIC_STDLIBS "Link solc against static versions of libgcc and libstdc++ on supported platforms" OFF)
 option(STRICT_Z3_VERSION "Use the latest version of Z3" ON)
+option(SOL_FORCE_RELEASE "Force the creation of a Release Version" OFF)
+option(SOL_VERSION_PRERELEASE "Getting the latest version of a prerelease" ON)
```

It's a string value. If you set it to `ON`, then that text will be included in the version string.
```suggestion
option(SOL_VERSION_PRERELEASE "Getting the latest version of a prerelease" "")
```